src/pages/ExperimentPerformance.jsx

```diff
@@ -526,7 +526,7 @@ const ExperimentPerformance = () => {
           <p>
             Visit{" "}
             <a
-              href="https://earth.bsc.es/gitlab/es/autosubmit_api/-/wikis/Performance-Metrics"
+              href="https://autosubmit-api.readthedocs.io/en/latest/guides/performance/index.html"
```

Comment:
I don't know about `jsx` pages, but in sphinx restructuredtext has a way to create links even between different projects (from what I recall). But we don't have to worry about that for now. +1 (sorry for approving too quickly before :sweat_smile: )

Reply:
No worries. I just found that the links were outdated and RtD was not correctly integrated :sweat_smile: 

We have to remember to update them again after Autosubmit is migrated to GitHub